<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopScreenSaverPreflet</class>
 <widget class="QWidget" name="DesktopScreenSaverPreflet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Background &amp; Screen Saver</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDesktop">
      <attribute name="title">
       <string>Desktop</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="preview">
           <property name="text">
            <string>&amp;Mode:</string>
           </property>
           <property name="buddy">
            <cstring>backgroundModeCombo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="backgroundModeCombo">
           <item>
            <property name="text">
             <string>Fit to Screen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fill Screen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stretch to Fit Screen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Center</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tile</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTreeView" name="categoriesTreeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QListView" name="backgroundsIconView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>128</width>
            <height>80</height>
           </size>
          </property>
          <property name="resizeMode">
           <enum>QListView::Adjust</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::Batched</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::IconMode</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="addButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-add-symbolic"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-remove-symbolic"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="changePictureCheckBox">
             <property name="text">
              <string>Change picture:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="changePictureComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Every 5 minutes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Every 10 minutes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Every 15 minutes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Every 30 minutes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Every hour</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Every day</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Every week</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Every month</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Every year</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="randomCheckBox">
           <property name="text">
            <string>Random order</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="translucentCheckBox">
           <property name="text">
            <string>Translucent menu bar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabScreenSaver">
      <attribute name="title">
       <string>Screen Saver</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
